### DISCUSSION TITLE
Can't find any data in ATTACK Navigator Playbook tab (Empty)

### DISCUSSION THREAD

**Discussion by ghefary** [Security-Onion-Solutions/securityonion#1826] [Unsupported Versions] [answered]
Hi there ,
The ATTACK Navigator didn't show me any data , also the Threat group and software dropdown are empty ,
can any one help tp solve this issue ??

**Comment by dougburks** [comment, marked as answer]
We're already tracking this in https://github.com/Security-Onion-Solutions/securityonion/issues/1798.
